docs/source/telegram.ext.basehandler.rst

```diff
@@ -0,0 +1,6 @@
+telegram.ext.BaseHandler
```

Comment:
if the file was renamed instead of deleted, the git blame history would be saved right?

Reply:
mh, I thought pycharms renaming thingy would use `git mv` …

Reply:
force pushed a commit where i manually used `git mv`, but that didn't help either. afterwards I also found https://stackoverflow.com/questions/2314652 :D